### PR TITLE
[0.17]  FEM: fixes

### PR DESCRIPTION
**Pull request by berndhahnebach**
- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)

FEM  fixes and in the regard of beam analysis.

### CODE REVIEW

File: src/Mod/Fem/femsolver/writerbase.py [hunk 1]
```diff
@@ -96,6 +101,25 @@ def get_constraints_fixed_nodes(self):
             # add nodes to constraint_conflict_nodes, needed by constraint plane rotation
             for node in femobj['Nodes']:
                 self.constraint_conflict_nodes.append(node)
+        # if mixed mesh with solids the node set needs to be splited because solid nodes do not have rotational degreee of freedom
```
Comment (luzpaz) [inline src/Mod/Fem/femsolver/writerbase.py]: > ...needs to be `split` because solid...

Comment (berndhahnebach) [reply src/Mod/Fem/femsolver/writerbase.py]: and degreeeeeeee ...

**Comment by berndhahnebach** [comment]
https://github.com/FreeCAD/FreeCAD/pull/1308/commits/84f641dab0c3f84d348683d2300c08aa648ebbe6#diff-4bb90d1a060d36ffa466f11e7a6bd04dR104

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**